### DISCUSSION TITLE
Cant find the Extensions-Tab

### DISCUSSION THREAD

**Discussion by AleB-Roni** [requarks/wiki#6383] [Help / Questions]
Hi,
I wanted to install 2 Extensions for my wiki. I read that the Extensions "Git" and "Sharp could help me with my copy & paste problem with pictures. I looked up the Wiki.js tutorial page, and it says that I can install the extensions via Administration-Area>Extensions
![image](https://user-images.githubusercontent.com/130990232/234004464-6e41ef06-727e-4dd1-948f-4f2997a5e90b.png)

I searched for the button on my Administration Area, but couldn't find it.
![image](https://user-images.githubusercontent.com/130990232/234004683-f165b0f1-e50b-493e-953b-19801e03b062.png)

I installed my wiki.js on a Windows device. I've tried to manually install "Sharp" but couldn't find (node_modules/sharp in my folder)

Thanks in advance :)

-Roni

**Comment by cdn0x12** [comment]
You're reading 3.x's documents which is not released at the moment. Documents for current version is located at https://docs.requarks.io/
